blockchain/chain_test.go

```diff
@@ -1311,3 +1313,381 @@ func TestIsAncestor(t *testing.T) {
 			branch2Nodes[0].hash.String())
 	}
 }
+
+func TestInvalidateBlock(t *testing.T) {
+	tests := []struct {
+		name     string
+		chainGen func() (*BlockChain, []*chainhash.Hash, func())
+	}{
+		{
+			name: "One branch, invalidate once",
```

Comment:
Should start with a lower case character for uniformity.

Reply:
Addressed in the latest pushes.